### PR TITLE
Update taglines

### LINKED ISSUE #1022: Unintended messaging in tagline
My apologies for a slightly pedantic bug report. Phoenix's tagline in the readme and website is:

> Productive. Reliable. Fast. Most web frameworks make you choose between speed and a productive environment. Phoenix gives you both.

If our set of choices is 3 (productive, reliable, and fast), and Phoenix explicitly gives us 2 ("speed" and "productive"), doesn't that imply that it isn't "reliable"? Especially because we're framing the question in a way that makes the user think about having to choose.

In IT, the ["given a set of 3, pick any 2"](http://whatis.techtarget.com/definition/pick-two) scenario is fairly common.

### PR DESCRIPTION
**Pull request by doomspork**
RFC @chrismccord / @josevalim - I've been following #1022 and thought I'd pounce on the opportunity to make the necessary changes.  This should close #1022 by updating `README.md`, `mix.exs`, and `index.html.eex` within the installer.

### CODE REVIEW

**Comment by josevalim** [comment]
Well, you should have waited as @chrismccord said, we are still not convinced on the tagline. :) So please hold on, we can ping on the issue once it is done for sure.

**Comment by doomspork** [comment]
:+1:

**Comment by chrismccord** [comment]
@doomspork we are going with 

> ### Productive. Reliable. Fast
> A productive web framework that
> does not compromise speed and maintainability

Can you update the PR?

**Comment by doomspork** [comment]
@chrismccord sure can, want me to squash the commits while I'm at it?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ![phoenix logo](https://raw.githubusercontent.com/phoenixframework/phoenix/master/priv/static/phoenix.png)
-> Productive. Reliable. Fast. Most web frameworks make you choose between speed and a productive environment. Phoenix gives you both.
+> ### Productive. Reliable. Fast.
+> A productive web framework that
+> does not compromise speed and maintainability
```
Comment (doomspork) [inline README.md]: @chrismccord this didn't line break like your comment, what fancy trick did you use?

Comment (chrismccord) [inline README.md]: it's fine without the break. Please squash

**Comment by doomspork** [comment]
That should do it, thanks @chrismccord!

**Comment by chrismccord** [comment]
<3 <3 <3

**Pull request merged**